ddtrace/tracer.py

```diff
@@ -28,6 +28,18 @@
 
 log = get_logger(__name__)
 
+debug_mode = asbool(environ.get("DD_TRACE_DEBUG", environ.get("DATADOG_TRACE_DEBUG", False)))
+
+if debug_mode and not hasHandlers(log):
```

Comment:
This feels like a risky change?

Reply:
Hmmm. It happens under very particular circumstances. DEBUG_MODE should only be used for debugging so I think it's a safe to do so. Did you have any particular concerns?

Reply:
I mean, what is going to happen is anyone using `ddtrace-run` who isn't configuring logging manually, is going to lose their logs.

No?

Reply:
Ah yes, only with `ddtrace-run` though. Unfortunately we've introduced that as something someone could be depending on. I think we'll just have to keep that support in `ddtrace-run` in addition to this code here.

Reply:
we can probably DRY up a bit, but yeah, we'll need to keep `logging.basicConfig()` for a bit :/